run_infrastructure.py

```diff
@@ -36,6 +36,7 @@ def run_infrastructure(
     run_tests: bool = False,  # Should we run the tests after creating the infra?
     run_create_superuser: bool = False,  # Should we run the create_superuser command?
     run_create_test_data: bool = False,  # Should we run the create_test_data command?
+    analytics_opt_out: bool = False, # Should we opt out of analytics?
```

Comment:
My formatter is adding another space here? 🤷‍♂️ 
```suggestion
    analytics_opt_out: bool = False,  # Should we opt out of analytics?
```